spec/features/volunteer_adds_a_case_contact_spec.rb

```diff
@@ -1,31 +1,46 @@
-require "rails_helper"
+require 'rails_helper'
 
-RSpec.feature "volunteer adds a case contact", type: :feature do
-  scenario "is successful" do
+RSpec.describe 'volunteer adds a case contact', type: :feature do
```

Comment:
Is there a reason you changed `feature` to `describe`? I don't think it makes a material difference but I'm just curious. 😄